File: Praticando Documentacao - Aula 01.08/numeros.py
import random
class Numeros:
    """_summary_
    Classe que possui serviços de manipulação de números 
    """
    @staticmethod
    def popular_aleatorio(lista, qtd, limite):
        """_summary_
            Metodo de classe que popula com numeros inteiros uma lista
        Args:
            lista (int): para numeros inteiros
            qtd (int): contem a quantidade de numeros a serem gerados na lista
            limite (int): range do numero
        """
        for i in range(0,qtd):
            lista.append(random.randint(0,limite))

File: Praticando Documentacao - Aula 01.08/Numeros.java
import java.util.Random;

/**
 * Classe com metodos estaticos para manipular listas de numeros
 */

public class Numeros {
    /**
     * metodo estatico que popula uma lista com numeros inteiros e aleatorios num range passado
     * @param lista - lista com numeros inteiros
     * @param qtd - quantidade de numeros a serem gerados na lista
     * @param limite - range no numero gerado
     */
    public static void popularAleatorio (ArrayList lista, int qtd, int limite);{
        Random gerador = new Random();
        for (int i = 0; i<qtd; i++){
            lista.add(gerador.nextInt(limite));
        }

    }
}
